src/util/htmlToJson.ts

```diff
@@ -200,9 +200,35 @@ const himalayaToBlock = (nodes: HimalayaNode[]): Block | Block[] => {
   return result
 }
 
+/** Gets all unique tag lists that is included in the text that we import. */
+const allTagsFinder = (nodes: Element[]): string[] => {
+  const tagLists = nodes.reduce((accum, current) => {
+    if (current?.children?.length) return [...accum, current.tagName, ...allTagsFinder(current.children as Element[])]
+    return current.tagName ? [...accum, current.tagName] : [...accum]
+  }, [] as string[])
+  return _.uniq(tagLists)
+}
+
+const tagsThatIsNotToBeStripped = [...allowedFormattingTags, 'span', 'li', 'ul']
+
+/** Generates dynamic regex expression. */
+const regexGenerator = (tags: string[]): RegExp => {
+  const regexString = tags
+    .reduce((accum, current) => {
+      return [...accum, `<${current}>`, `</${current}>`]
+    }, [] as string[])
+    .join('|')
```

Comment:
I think this is equivalent, and simpler: `` `</?${tags.join('|')}>` ``

Reply:
True! Thanks for the suggestion :)

Reply:
I don't think this will work when there are 'n' number of tags in text.

Reply:
https://regexr.com/6h79j